src/foam/u2/view/FObjectView.js

```diff
@@ -39,34 +47,44 @@ foam.CLASS({
         }
       }
     },
-    'choices'
-  ],
+    'of',
+    {
+      name: 'choices',
+      expression: function(of) {
+        var models = Object.keys(foam.USED)
```

Comment:
This is pretty inefficient and I'm not sure if there's a better place to get all models that have been created. Is there a better way to walk all of our models?

Reply:
Would be more _efficient_ to cache this list and invalidate the cache every time a new model is loaded, but i'm not sure that's worth the effort.

Reply:
Yeah I was thinking the same thing. I was chatting with Kevin earlier and he suggested using foam.UNUSED too which I was afraid to do originally but it would be easy to add. Does anybody object to me going through all the UNUSED models too? It'll effectively flip them all to the USED pile though.